src/blueprints/ember-addon/package.json

```diff
@@ -23,7 +23,7 @@
     "concurrently": "<%= context.projectRoot.devDependencies['concurrently'] %>"
   }
 }<% } else if (options.packageManager.isPnpm) { %>{
-  "name": "<%= options.packages.addon.name %>",
+  "name": "workspace-root",
```

Comment:
By giving the workspace root a name, we may easily distinguish where dependencies come from, when we run the `outdated` command, e.g. `pnpm outdated -r`.